scripts/pre_commit_linter.py

```diff
@@ -3124,26 +478,44 @@ def main(args=None):
 
     read_files(all_filepaths)
     categorize_files(all_filepaths)
-    linting_processes, result_queues, result_stdout = _lint_all_files(
-        _FILES['.js'], _FILES['.ts'], _FILES['.py'], _FILES['.html'],
-        _FILES['.css'], verbose_mode_enabled)
-    code_owner_message = _check_codeowner_file(verbose_mode_enabled)
-    # Pylint requires to provide paramter "this_bases" and "d", guess due to
-    # meta class.
-    js_ts_lint_checks_manager = JsTsLintChecksManager( # pylint: disable=no-value-for-parameter
-        verbose_mode_enabled)
-    other_lint_checks_manager = OtherLintChecksManager(   # pylint: disable=no-value-for-parameter
+
+    max_concurrent_runs = 25
```

Comment:
@kevinlee12  If I set this to 1 then the linting process becomes very slow and never ends.